packages/documentation/src/stories/components/forms/card-control/standard-html/card-control-firefox-fallback.sample.ts

```diff
@@ -0,0 +1,12 @@
+document.addEventListener('input', e => {
```

Comment:
Just by curiosity, did you move the file or recreate it and copy and paste the content or with another method? Github sees it as a new file, not as a move.